pkg/virt-handler/vm.go

```diff
@@ -1026,6 +1026,8 @@ func (c *VirtualMachineController) Run(threadiness int, stopCh chan struct{}) {
 	go c.gracefulShutdownInformer.Run(stopCh)
 	cache.WaitForCacheSync(stopCh, c.domainInformer.HasSynced, c.vmiSourceInformer.HasSynced, c.vmiTargetInformer.HasSynced, c.gracefulShutdownInformer.HasSynced)
 
+	// set log verbosity
+	c.clusterConfig.SetConfigModifiedCallback(c.shouldChangeLogVerbosity)
```

Comment:
what would you think about placing this in the `cmd/virt-handler/virt-handler.go` file, maybe somewhere around here https://github.com/kubevirt/kubevirt/blob/b20eb8511f6182ae9a3c44665a42e3c436306145/cmd/virt-handler/virt-handler.go#L300 ?

We only have one controller in virt-handler right now. I don't know if that will always be the place. Seems like this verbosity logic would fit better outside of one of the individual controller packages.

Reply:
Yep, sounds good. I'll move it there.

Reply:
Done